.travis.yml

```diff
@@ -6,11 +6,3 @@ node_js:
 - '0.10'
 script:
 - npm run build
-deploy:
```

Comment:
@jeremyruppel said to kill it since we deploy manually. https://github.com/flickr/yakbak/commit/27eab62940c2d14f64789804ddadf77094fd91a6